### ISSUE TITLE
Maven plugin logs wrong violation count

### ISSUE THREAD

**Issue by wied03** [avioconsulting/mule-linter#84]
In the branch with the Maven plugin, the rule count is used for violations,

ConsoleFormatter.java
```java
        log.info( String.format("Found a total of %s violations of %s rules.", rulesCount,rulesCount));
```

Therefore you get a message (but not a failure) stating there were violations when there were not:

```
[INFO] ****************************************************************************
[INFO] AVIO Mule linter execution results
[INFO] 9 rules executed.
[INFO] Rule validation results
[INFO] Found a total of 9 violations of 9 rules.
[INFO] ****************************************************************************
```
